contributing/community/releases.rst

```diff
@@ -19,14 +19,21 @@ published through a *time-based model*:
 
 .. tip::
 
-    `Subscribe to Symfony Roadmap notifications`_ to receive an email when a new
+    `Subscribe to Symfony Release notifications`_ to receive an email when a new
     Symfony version is published or when a Symfony version reaches its end of life.
 
 .. _contributing-release-development:
 
 Development
 -----------
 
+.. note::
+
+    The Symfony project is an open-source community driven development framework.
+    There is no roadmap written or defined in advance. Every feature request
+    may or may not be developed in future version based on the community.
+    Symfony core team members can help move things forward if the intereset is here.
+
```

Comment:
community driven => community-driven (to check)
in future version => in a future version (or in future versions)
intereset => interest

Reply:
Thanks for the review. I did this changes while merging the PR.